www/src/components/showcase-details.js

```diff
@@ -143,6 +145,10 @@ function ShowcaseModal({ children, location, isModal }) {
   if (!isModal) return children
   const { previousSite, nextSite } = usePrevAndNextSite(location.pathname)
   const { filters } = location.state || {}
+  const { previousSite, nextSite } = usePrevAndNextSite(
+    location.pathname,
+    filters
+  )
```

Comment:
This must lead to a syntax error: the same bindings are defined multiple times. Though I guess this is the result of trying to resolve a merge conflict. You probalby intended to drop line 146 here.

Reply:
Ah, good catch.